### DISCUSSION TITLE
Add equation numbers in document templates on the typst website

### DISCUSSION THREAD

**Discussion by hasantahir** [typst/typst#2855] [General]
On the [typst website](https://typst.app/assets/images/oxford-large.png), the IEEE, Oxford and MDPI templates should have equation numbers for completeness. 

<img width="809" alt="SCR-20231204-phjl" src="https://github.com/typst/typst/assets/11236148/82c79613-444b-4aae-bc79-5d3164407b8e">

**Comment by Enivex** [comment]
Equation numbers are usually only present if the equation is referred to in the text.

**Reply by yochem** [reply]
For IEEE not true according to this SE answer (and the source they cite) https://academia.stackexchange.com/a/82320.

**Comment by yochem** [comment]
The examples are probably old, since the IEEE template the typst web IDE uses includes equation numbers.
